lib/vm/src/memory.rs

```diff
@@ -316,8 +331,26 @@ impl LinearMemory for VMOwnedMemory {
     fn try_clone(&self) -> Option<Box<dyn LinearMemory + 'static>> {
         None
     }
+
+    /// Copies this memory to a new memory
+    fn fork(&mut self) -> Result<Box<dyn LinearMemory + 'static>, MemoryError> {
```

Comment:
Let's use clone. I think is a more accurate naming

Reply:
Renamed to `duplicate()`.